apps/sms/views/conversation/js/conversation.js

```diff
@@ -1876,9 +1917,13 @@ var ConversationView = {
     }
     params.items.push(subjectItem);
 
-    // If we are on a thread, we can call to SelectMessages
+    // If we are on a thread, we can call to SelectMessages and add recipients
     if (Navigation.isCurrentPanel('thread')) {
       params.items.push({
+        l10nId: 'addRecipients-label',
+        method: this.addRecipients.bind(this)
```

Comment:
Oh that's funny. github removes the "obsolete" marker when I restore a previous diff :-) I thought they would stay obsolete...